.eslintrc.js

```diff
@@ -1,30 +1,54 @@
 module.exports = {
-  'extends': 'eslint:recommended',
-    'parserOptions': {
-      'sourceType': 'module',
-      'ecmaVersion': 2017
+  'extends': [
+    'airbnb-base/legacy',
+  ],
+  'parserOptions': {
+    'sourceType': 'module',
+    'ecmaVersion': 2017,
   },
   'env': {
     'node': true,
     'es6': true,
     'mocha': true,
-    'browser': true
+    'browser': true,
   },
-  'rules': {
-    'indent': ['error', 2, { 'SwitchCase': 1 }],
-    'quotes': ['error', 'single', { 'allowTemplateLiterals': true }],
-    'semi': [2, 'always'],
-    'brace-style': ['error', '1tbs'],
-    'comma-style': ['error', 'last'],
-    'comma-spacing': ['error', {'after': true, 'before': false}],
-    'eol-last': ['error', 'always'],
-    'func-call-spacing': ['error', 'never'],
-    'semi-spacing': ['error', {'before': false, 'after': true}],
-    'keyword-spacing': ['error', {'before': true, 'after': true}],
+  "rules": {
+    // The one assertion of personal preference: no spaces before parentheses
```

Comment:
Oh no, not this again. Accepting common standards is hard, but it requires some courage and patience. It worth it.

Reply:
I have comments explaining why I have disabled each of the rules. I did not do it without a good reason. And this top rule is the only one that I actually changed.

Reply:
You might find [this issue](https://github.com/airbnb/javascript/issues/1089) instructive. The people that maintain the airbnb style guide recognize that some rules are more controversial than others and that there are certain cases where disabling specific rules makes more sense than consistency for its own sake.

Reply:
> I did not do it without a good reason. And this top rule is the only one that I actually changed.

I can find a good reason for each rule why not to disable it. It is still a matter of personal preference, which plagues JS community. That's why striker tools, like Standard or Prettier, probably is a better choice. I'm just slightly disappointed that Nunjucks will continue that charade.

Anyway, arguing about codestyle is the same as arguing about tabs vs spaces. I've declared my concerns and they have been heard, I don't have anything meaningful to add. The choice is yours anyway to make.